nvflare/app_common/pt/learners/supervised_learner.py

```diff
@@ -124,11 +123,6 @@ def train_config(self, fl_ctx: FLContext, train_config_file_path: str):
         with open(train_config_file_path) as file:
             config_info = json.load(file)
 
-        self.fedproxloss_mu = config_info.get("fedproxloss_mu", 0.0)
-        if self.fedproxloss_mu > 0:
-            self.log_info(fl_ctx, f"Using FedProx loss with mu {self.fedproxloss_mu}")
-            self.criterion_prox = PTFedProxLoss(mu=self.fedproxloss_mu)
-
         self._extra_train_config(fl_ctx, config_info)
```

Comment:
The file name and the class name (supervised_learner and supervised_ditto_learner) suggests this should be general purpose.

However from the implementation, I see the users need to define the self.model (and some other stuff) somewhere for this to work. (like getting stuff from config), and the MONAI text in docstring suggest these are not general purpose.

We either need to fix the issue in this PR, or we don't move into app_common for now, we do a separate PR to move them.

Reply:
sure let's reverse back and have some further discussions for a separate PR later